packages/components/src/autocomplete/autocompleter-ui.native.js

```diff
@@ -49,7 +50,11 @@ export function getAutoCompleterUI( autocompleter ) {
 		value,
 		reset,
 	} ) {
-		const [ items ] = useItems( filterValue );
+		const [ allItems ] = useItems( filterValue );
+		const items = useMemo(
```

Comment:
How did you know to use `useMemo` here? Is it because the Component is animating that we need it?

Reply:
Since we are filtering the list as soon as the user is typing it was causing a render loop, so adding `useMemo` prevents unnecessary renders unless `allItems` changes.